### DISCUSSION TITLE
Issues with turning on devices

### DISCUSSION THREAD

**Discussion by elijahshumway** [webdjoe/pyvesync#237] [Announcements]
I'm able to see my device info, but am getting invalid request parameters when trying to turn on or off. Did something with the API parameters/calls change that aren't reflected here? Not sure what I'm doing wrong

**Comment by webdjoe** [comment]
What device do you have? What is the error message you are receiving?

**Comment by webdjoe** [comment]
Please open a new issue if you are still having this problem
